src/analytics/Trace.tsx

```diff
@@ -68,9 +68,8 @@ export const Trace = memo(
           git_commit_hash: analyticsConfig?.commitHash,
         })
       }
-      // Impressions should only be logged on mount.
       // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
You should still have a comment explaining why eslint is disabled.

```suggestion
      // Impressions should only be logged on mount or is `shouldLogImpression` toggles
      // (ie if the component becomes "viewed"), but not if/when other deps update.
      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Reply:
I was updated to reflect on the component definition comment but also added it here. Thanks for pointing it out!